.github/labeler.yml

```diff
@@ -0,0 +1,273 @@
+### labeler.yaml
+### This is a best naive guess on auto labeling
+### with this script: https://repl.it/@maxandersen/labels
+### should be verified/tuned by changing the script and once
+### committed updates would be done manually.
+
+
+### Non-covered dirs....
+### - extensions
+### - extensions/agroal
+### - extensions/amazon-dynamodb
+### - extensions/azure-functions-http
+### - extensions/caffeine
+### - extensions/flyway
+### - extensions/jackson
+### - extensions/jdbc
+### - extensions/jgit
+### - extensions/jsch
+### - extensions/jsonb
+### - extensions/jsonp
+### - extensions/keycloak-authorization
+### - extensions/mailer
+### - extensions/netty
+### - extensions/pom.xml
+### - extensions/quartz
+### - integration-tests
+### - integration-tests/.project
+### - integration-tests/.settings
+### - integration-tests/amazon-dynamodb
+### - integration-tests/class-transformer
+### - integration-tests/flyway
+### - integration-tests/jackson
+### - integration-tests/jgit
+### - integration-tests/jsch
+### - integration-tests/jsonb
+### - integration-tests/keycloak
+### - integration-tests/keycloak-authorization
+### - integration-tests/main
+### - integration-tests/pom.xml
+### - integration-tests/quartz
+### - integration-tests/shared-library
+### - integration-tests/test-extension
+### - integration-tests/virtual-http
+
+### Non-covered area/
+### area/bootstrap-maven
+### area/build
+### area/camel
+### area/core
+### area/dependencies
+### area/dev-mode
+### area/gradle
+### area/housekeeping
+### area/infra
+### area/new-http
+### area/oidc-interoperability
+### area/openapi
+### area/platform
+### area/reactive-messaging
```

Comment:
This one should be used for the `smallrye-reactive-messaging*` extensions.

Reply:
done.